blog/docs/releases/announcing-knative-v1-1-release.md

```diff
@@ -0,0 +1,166 @@
+
+Follow the instructions in the documentation
+[Installing Knative](https://knative.dev/docs/admin/install/) for the respective component.
+
+#### Table of Contents
+- [Highlights](#highlights)
+- [Serving v1.1](#serving-v11)
+- [Eventing v1.1](#eventing-v11)
+- [Eventing Extensions](#eventing-extensions)
+    - [Apache Kafka Broker v1.1](#apache-kafka-broker-v11)
+    - [RabbitMQ Broker and Source v1.1](#rabbitmq-broker-and-source-v11)
+- `kn` [CLI v1.1](#client-v11)
+- [Knative Operator v1.1](#operator-v11)
+- [Thank you contributors](#thank-you-contributors)
+
+
+### Highlights
+
+- Serving now has a global `min-scale` configuration
+- Eventing brings more experimental features
+
+
+### Serving v1.1
+
+<!-- Original notes are here: https://github.com/knative/serving/releases/tag/knative-v1.1.0 -->
+
+#### 💫 New Features & Changes
+
+- Add cluster wide default min-scale ([#12290](https://github.com/knative/serving/pull/12290))
+- HPA autoscaler stabilization window can be set from autoscaling window annotation ([#12286]](https://github.com/knative/serving/pull/12286))
+- Metrics with names other than "cpu" or "memory" are allowed as pod custom metrics. ([#12277](https://github.com/knative/serving/pull/12277))
+
+#### 🐞 Bug Fixes
+
+- Allow setting TCP and HTTP port to be equal to containerPort on readiness and liveness probes. ([#12225](https://github.com/knative/serving/pull/12225)
+- Drops the unix socket listener from Queue Proxy, since it is no longer used ([#12298](https://github.com/knative/serving/pull/12298)
+- Generated HPAs are now v2beta2.
+    - Window annotation will be set to HPA stabilization window ([#12278](https://github.com/knative/serving/pull/12278)
+- Remove performance tests using Mako ([#12266](https://github.com/knative/serving/pull/12266)
+- The activator optimisation which directly probes the queue proxy for readiness rather than waiting for Kubernetes to report readiness is now disabled when exec probes are used (since queue proxy cannot execute these probes on the user container's behalf). ([#12250](https://github.com/knative/serving/pull/12250)
+- Use pkg/drain in queue proxy ([#12033](https://github.com/knative/serving/pull/12033)
+
+### Eventing v1.1
+
+<!-- Original notes are here: https://github.com/knative/eventing/releases/tag/knative-v1.1.0 -->
+
+#### 💫 New Features & Changes
+
+- New experimental-feature "delivery-retryafter" flag allows use of "DeliverySpec.retryAfter" to configure handling of Retry-After headers in 429 / 503 responses. See https://github.com/knative/docs/blob/main/docs/eventing/experimental-features.md ([#5813](https://github.com/knative/eventing/pull/5813)
+- All core Knative Eventing Pods should now be able to run in the restricted pod security standard profile. ([#5863](https://github.com/knative/eventing/pull/5863)
+- Triggers now include a CloudEvents Subscriptions API compatible filters field as an experimental feature. ([#5715](https://github.com/knative/eventing/pull/5715)
```

Comment:
Each line here needs a closing `)`

Reply:
Good catch an all this formatting items 🙏